symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/jwt/JwtHelper.java

```diff
@@ -42,8 +42,8 @@
 @API(status = API.Status.INTERNAL)
 public class JwtHelper {
 
-  // Expiration of the jwt
-  public static final Long JWT_EXPIRATION_MILLIS = 300_000L;
+  // Expiration of the jwt, 5 minutes maximum, use a little less than that in case of clock differents
```

Comment:
I would say "in case of different clocks" or "in case of different clock skews" ?

Reply:
Indeed I meant clock differences, I fixed it 👍